src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlMemberTranslatorProvider.cs

```diff
@@ -40,10 +41,11 @@ public NpgsqlMemberTranslatorProvider(
                 new NpgsqlDateTimeMemberTranslator(typeMappingSource, sqlExpressionFactory),
                 new NpgsqlJsonDomTranslator(typeMappingSource, sqlExpressionFactory, model),
                 new NpgsqlLTreeTranslator(typeMappingSource, sqlExpressionFactory, model),
+                new DictionaryTranslator(typeMappingSource, sqlExpressionFactory, model),
```

Comment:
Needs to be before the JsonPocoTranslator as it needs to intercept methods on Dictionary types that are json types.